Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -392,6 +393,42 @@ public ConnectionMode ConnectionMode
         /// <seealso cref="ItemRequestOptions.EnableContentResponseOnWrite"/>
         /// <seealso cref="TransactionalBatchItemRequestOptions.EnableContentResponseOnWrite"/>
         public bool? EnableContentResponseOnWrite { get; set; }
+
+        /// <summary>
+        /// Gets or sets the <see cref="System.Text.Json.JsonSerializerOptions"/> for the default STJ serializer <see cref="CosmosSystemTextJsonSerializer"/>.
+        /// Note that if this option is provided, then the SDK will use the <see cref="CosmosSystemTextJsonSerializer"/> as the default serializer and set
+        /// the serializer options as the constructor args.
+        /// </summary>
+        /// <example>
+        /// An example on how to configure the STJ serialization option to ignore null values
```

Comment:
```suggestion
        /// An example on how to configure the System.Text.Json serialization option to ignore null values
```

We should probably just avoid saying STJ anywhere in the public surface area.

Reply:
Updated.